example-playbooks/replace_node/replace_node.yml

```diff
@@ -127,12 +148,31 @@
     - ansible-scylla-node
 
 
+# Update scylla-monitoring before the replacement starts to be able to see the
+# streaming progress
 - name: Update scylla-monitoring
   hosts: scylla-monitor
   roles:
     - ansible-scylla-monitoring
 
 
+# We want to silence alerts for the new node until the end of the replacement
+# The duration will be "new_node_bootstrap_wait_time_sec * 2", since this should
+# be enough for waiting for CQL port to become available and for waiting for the
+# node to become healthy.
+- name: Silence alerts for the new node
```

Comment:
Does Monitoring restart clears previously set alerts silences? We do preserve the Prometheus "data". Doesn't it persists silences on the disk?

Reply:
Yes, it does persist silences.